ast/src/lib.rs

```diff
@@ -18,28 +18,47 @@ pub enum Keyword {
     Else,
     Export,
     Extends,
-    False,
     Finally,
     For,
     Function,
+    Get,
     If,
     In,
     InstanceOf,
     Import,
     Let,
     New,
-    Null,
     Of,
     Return,
+    Set,
     Super,
+    Static,
     Switch,
     This,
     Throw,
-    True,
     Try,
     TypeOf,
     Var,
     Void,
     While,
     Yield,
 }
+
+#[derive(Clone, Copy, PartialEq, Eq, Debug, Serialize, Deserialize)]
+pub enum Reserved {
+    False,
+    Null,
+    True,
+
+    // future reserved words
+    Enum,
+    Implements,
+    Interface,
+    Package,
+    Private,
+    Protected,
+    Public,
+
+    // deprecated reserved words
+    With,
```

Comment:
This isn't deprecated – it's just denied in strict mode, and I expect it would be a keyword. Please confirm.

Reply:
@load1n9 Did you confirm with the specification?

Reply:
We talked about how we really didn't want it but for now we could keep it

Reply:
May I ask where this talk was? Why wouldn't we want it?

Reply:
@Phosra https://discord.com/channels/1012084654389604372/1012084654389604376/1081302847930499202
First message of the convo

Reply:
Right. I disagreed at https://discord.com/channels/1012084654389604372/1012084654389604376/1081334881642094612.
No issue here, it's to be kept until we decide on something different later.